tox.ini

```diff
@@ -0,0 +1,48 @@
+[tox]
+isolated_build = true
+envlist = py37,lint,docs
+
+[travis]
+python =
+    3.7: py37
+
+[testenv]
+passenv = SSH_AUTH_SOCK
+whitelist_externals = poetry
+setenv =
+    PYTHONPATH = {toxinidir}
+commands =
+    poetry install -v
+    poetry run pytest --ignore experiments
+
+[testenv:test1]
+commands =
+    poetry install -v
+    poetry run pytest --ignore experiments --cov-append csrank/tests/test_choice_functions.py
+
+[testenv:test2]
+commands =
+    poetry install -v
+    poetry run pytest --ignore experiments --cov-append csrank/tests/test_discrete_choice.py
+
+[testenv:test3]
+commands =
+    poetry install -v
+    poetry run pytest --ignore experiments --cov-append csrank/tests/test_ranking.py
+
+[testenv:test4]
+commands =
+    poetry install -v
+    poetry run pytest --ignore experiments --cov-append csrank/tests/test_fate.py csrank/tests/test_losses.py csrank/tests/test_metrics.py csrank/tests/test_util.py csrank/tests/test_callbacks.py
+
+[testenv:docs]
+changedir = docs
+commands =
+    poetry install -v
+    sphinx-build -d {envtmpdir}/doctrees . {envtmpdir}/html
+
+[testenv:lint]
+basepython = python3.7
```

Comment:
Python 3.7 is specified several times. Is that necessary?

Reply:
Thinking about this a bit more, we shouldn't let perfect be the enemy of good enough here. It would probably take a bit of additional time to unify the python version specification, we should get some version of tox support in first.